index.html

```diff
@@ -28,3 +29,53 @@
         </div>
     </div>
 </div>
+
+{% raw  %}
+<script type="text/javascript">
```

Comment:
It might be nice to have this in a separate `.js` file.

Reply:
I moved most of the js code to `js/meetup-events.js` but left the trigger code here.